haros/haros.py

```diff
@@ -375,8 +384,9 @@ def _generate_dir(self, path, dir_dict, overwrite=True):
 class HarosRunner(object):
     """This is a base class for the specific commands that HAROS provides."""
 
-    def __init__(self, haros_dir, log, run_from_source):
+    def __init__(self, haros_dir, config_dir, log, run_from_source):
```

Comment:
This parameter name is confusing. It is named as if you were expecting the directory containing the config file, but you are actually expecting the path to the file itself. Suggested: `config_path`